migrations/20210519172140_v1.11.0.js

```diff
@@ -0,0 +1,18 @@
+const fs = require('fs');
+const sql = fs.readFileSync(__filename + '.sql', 'utf8');
+
+exports.up = async (db) => {
+	await db.schema.alterTable('package_version', (table) => {
+		table.dropForeign([ 'packageId' ]);
+		table.dropUnique([ 'packageId', 'version' ]);
+
+		table.string('type', 16).notNullable().defaultTo('version');
+
+		table.unique([ 'packageId', 'version', 'type' ]);
+		table.foreign('packageId').references('id').inTable('package').onUpdate('cascade').onDelete('cascade');
```

Comment:
Because there was a key, that couldn't be deleted without removing the foreign key constraint.
Anyway, only the new column is added in the latest commit, and keys and relations left unmodified.